Rolique/UI/Profile/ProfileDetail/ProfileDetailViewController.swift

```diff
@@ -256,8 +255,7 @@ final class ProfileDetailViewController<T: ProfileDetailViewModel>: ViewControll
   }
   
   @objc func didSelectSlackButton(sender: UIButton) {
-    guard let id = viewModel.user?.id else { return }
-    self.openSlack(with: id)
+    _ = viewModel.user?.id.map({ self.openSlack(with: String($0)) })
```

Comment:
Not crucial, but on future, there is method `do` which wraps `map` and returns `Void`